src/main/java/com/solvd/carina_testing/pages/CheckoutPage.java

```diff
@@ -0,0 +1,14 @@
+package com.solvd.carina_testing.pages;
+
+import com.zebrunner.carina.webdriver.decorator.PageOpeningStrategy;
+import com.zebrunner.carina.webdriver.gui.AbstractPage;
+import org.openqa.selenium.WebDriver;
+
+public class CheckoutPage extends AbstractPage {
+
+    public CheckoutPage(WebDriver driver) {
+        super(driver);
+        setPageOpeningStrategy(PageOpeningStrategy.BY_URL);
```

Comment:
I would better use strategy by Element, sometimes url can be correct but the page wasn't loaded, or just white screen, or alert with error